build-info/docker/image-info.dotnet-dotnet-docker-nightly.json

```diff
@@ -1,2575 +1,2172 @@
-          "arm64v8/buildpack-deps:focal-curl": "arm64v8/buildpack-deps@sha256:ed9c12af416d915dc120820658d31316ea718277acfc04ae7a6e9277320bb0fc"
-        },
-        "simpleTags": [
-          "5.0-focal-arm64v8",
-          "5.0.0-preview-focal-arm64v8"
-        ],
-        "digest": "sha256:625522e42085ef937a21eb862244b64279c40f43ee0099e158ae3b5618c5e900"
-      },
-      "5.0/aspnet/nanoserver-1809/amd64/Dockerfile": {
-        "baseImages": {
-          "mcr.microsoft.com/windows/servercore:1809": "mcr.microsoft.com/windows/servercore@sha256:9d0dff5e064c87d612503e1559c3d14353b404a1cd4b53ac1373e0fc62017971"
-        },
-        "simpleTags": [
-          "5.0.0-preview-nanoserver-1809",
-          "5.0-nanoserver-1809"
-        ],
-        "digest": "sha256:903eef3902d89c081d05b9dbbd14577836063b9833c350ade340305b8a29b390"
-      },
-      "5.0/aspnet/nanoserver-1809/arm32v7/Dockerfile": {
-        "simpleTags": [
-          "5.0-nanoserver-1809-arm32v7",
-          "5.0.0-preview-nanoserver-1809-arm32v7"
-        ],
-        "digest": "sha256:3b407f167a79693c92b87feac230860d482e7a8f2e898969e105962667dc4a43"
-      },
-      "5.0/aspnet/nanoserver-1903/amd64/Dockerfile": {
-        "baseImages": {
-          "mcr.microsoft.com/windows/servercore:1903": "mcr.microsoft.com/windows/servercore@sha256:ad1d822527c131fa1939dc25083a91c3bb62640b60e754b9f7e504d28cde5157"
-        },
-        "simpleTags": [
-          "5.0.0-preview-nanoserver-1903",
-          "5.0-nanoserver-1903"
-        ],
-        "digest": "sha256:54a62fa1f0a8c17624813fb4ae40158a7217e176d9dac5d05c9d032a1c8ab83f"
-      },
-      "5.0/aspnet/nanoserver-1909/amd64/Dockerfile": {
-        "baseImages": {
-          "mcr.microsoft.com/windows/servercore:1909": "mcr.microsoft.com/windows/servercore@sha256:d0a93cf38daadc18ca7b2ed537bde74cfb867f50556d44e8adcfec4a1d1f2de4"
-        },
-        "simpleTags": [
-          "5.0.0-preview-nanoserver-1909",
-          "5.0-nanoserver-1909"
-        ],
-        "digest": "sha256:e71d3f1abb80155ce8618512bdddc9aa65b2485ef8bd22b514b7e8c140f83436"
-      }
+{
+  "schemaVersion": "1.0",
+  "repos": [
+    {
+      "repo": "dotnet/core-nightly/aspnet",
+      "images": [
+        {
+          "manifest": {
+            "digest": "sha256:db7530372e4b0073a560e28c57652084520249a366845eb2d735839852ecf8f8",
+            "created": "2020-03-23T13:22:39.0984864Z",
+            "sharedTags": [
+              "2.1.16",
+              "2.1"
+            ]
+          },
+          "platforms": [
+            {
+              "dockerfile": "2.1/aspnet/nanoserver-1809/amd64/Dockerfile",
+              "simpleTags": [
+                "2.1.16-nanoserver-1809",
+                "2.1-nanoserver-1809"
+              ],
+              "digest": "sha256:b23288ea3e92a12a584b8279b37dd5e7eb914dd29fd6a5d5bc7211b5248c9ecc",
+              "baseImageDigest": "mcr.microsoft.com/windows/nanoserver@sha256:5de6bd32bd453d60c8f549d28845552e89ad3652566e141ac82023b6ba10374d",
```

Comment:
This is wrong.  I noticed it is wrong in the existing data as well.  It looks like the logic in the image-info-schema branch will correct this.  I don't think anything needs to be done here but now that info will include internal references, does the hello build kitty infra ignore "internal" base images?  By that I mean, it shouldn't retrieve the internal image digests from the registries. There is no need to monitor them.

Reply:
What is wrong about this?  This isn't an in internal base image; it's the base Windows image.

Reply:
My mistake, I got mixed up thinking about `installer` stage.  I think my question about hello build kitty applies.  It looks like now we always include the base image where as before we only included external base images.

Reply:
The `BaseImageDigest` value for a base image that is internally built is not currently stored.  This is because the base image digest is retrieved before the images have been pushed, so it ends up being null.  I've logged https://github.com/dotnet/docker-tools/issues/447 to track this.